core/src/luigi-config/customResources.js

```diff
@@ -58,35 +58,22 @@ async function loadTargetClusterCRs(authData) {
     console.warn('Cannot load target cluster CRs', e);
   }
   return (items || [])
-    .map(item => {
-      const cr = Object.entries(item?.data || []).reduce(
-        (acc, [key, value]) => {
-          try {
-            const match = key.match(/^translations(-([a-z]{2}))?$/);
-            if (match) {
-              let translations = acc.translations || {};
-              const lang = match[2];
-              const langTranslations = jsyaml.load(value);
-              if (lang) {
-                translations = merge(translations, {
-                  [lang]: langTranslations,
-                });
-              } else {
-                translations = merge(translations, langTranslations);
-              }
-              return { ...acc, translations };
+    .map(configMap => {
+      const cr = Object.fromEntries(
+        Object.entries(configMap?.data || []).map(
```

Comment:
```suggestion
        Object.entries(configMap?.data || {}).map(
```
`configMap.data` cannot be an array.

Reply:
I mean, `Object.entries([]) `is the same as `Object.entries({})`...